Contained app/Contained app/GameViewController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  GameViewController.swift
+//  Contained app
+//
+//  Created by Drew Miller on 5/18/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import SpriteKit
+
+class GameViewController: UIViewController {
+
+    @IBOutlet weak var skview: SKView!
```

Comment:
You are getting an error saying that is essentially saying there's no property in the class name skyview. So the compiler is looking for something named skyview.